src/main/java/org/spdx/library/conversion/Spdx2to3Converter.java

```diff
@@ -894,14 +995,51 @@ public ListedLicenseException convertAndStore(org.spdx.library.model.v2.license.
 		return toListedException;
 	}
 
+	/**
+	 * Converts an SPDX spec version 2 SPDX AnyLicenseIfno to an SPDX spec version 3 LicenseExpression
+	 * @param fromLicense an SPDX spec version 2 AnyLicenseIfno
```

Comment:
Same typo here too.

Reply:
Thanks for the catches @pmonks - I'll add a fix